drivers/watchdog/imx_watchdog.c

```diff
@@ -44,7 +44,7 @@ void __attribute__((weak)) reset_cpu(ulong addr)
 {
 	struct watchdog_regs *wdog = (struct watchdog_regs *)WDOG1_BASE_ADDR;
 
-	imx_watchdog_expire_now(wdog, true);
+	imx_watchdog_expire_now(wdog, false);
```

Comment:
@ricardosalveti @igoropaniuk @Tim-Anderson this change comes from nxp's https://source.codeaurora.org/external/imx/uboot-imx/commit/?h=imx_v2020.04_5.4.70_2.3.0&id=44f5949dd94317c918c23854413ac30b7feceaf0
not sure if this would impact any functionality that you are working on